PSI Food waste/Startup.cs

```diff
@@ -23,13 +23,24 @@ public Startup(IConfiguration configuration)
             Configuration = configuration;
         }
 
+        //public IProductRepository _productRepository;
+
+        //public Startup(IConfiguration configuration, IProductRepository productRepository)
+        //{
+        //    Configuration = configuration;
+        //    _productRepository = productRepository;
+        //}
+
         public IConfiguration Configuration { get; }
 
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddRazorPages();
             services.AddSession();
+            services.AddSingleton<IEmployeeRepository, MockEmployeeRepository>();
```

Comment:
These interfaces dont exist so program doesn't compile

Reply:
Yes, I totally forgot to delete them after testing